Source/Shared/Storage/DiskStorage.swift

```diff
@@ -307,16 +307,7 @@ public final class DiskStorage: StorageAware {
    - Returns: A md5 or base64 string
    */
   func fileName(_ key: String) -> String {
-    if let digest = key.data(using: String.Encoding.utf8)?.md5() {
-      var string = ""
-      for byte in digest {
-        string += String(format:"%02x", byte)
-      }
-
-      return string
-    } else {
-      return key.base64()
```

Comment:
Yeah, I will. This if/else with `md5` and `base64` will make it inconsistent. Better to stick with 1 only